### PR TITLE
adding h_slt corrmat and setting up scr_h skeleton package

### PR DESCRIPTION
**Pull request by pdavidsonFIA**

### CODE REVIEW

File: solvency2sf/corr_h_slt.csv [hunk 1]
```diff
@@ -0,0 +1,7 @@
+corr_h_slt,mortality,longevity,morbidity,expense,revision,lapse
+mortality,1.00,0.25,0.25,0.25,0.00,0.00
+longevity,0.25,1.00,0.00,0.25,0.25,0.25
```
Comment (jeanm-m) [inline solvency2sf/corr_h_slt.csv]: shouldn't correlation mortality vs longevity be negative, -0.25 instead of 0.25?

**Review by jeanm-m — CHANGES_REQUESTED** [review]
thanks a lot for adding that super fast, I can use it right away! I just have a doubt about the sign of a correlation

File: solvency2sf/corr_h_slt.csv (hunk 1 above)
Comment (pdavidsonFIA) [reply solvency2sf/corr_h_slt.csv]: good spot!

Comment (pdavidsonFIA) [reply solvency2sf/corr_h_slt.csv]: @jeanm-m latest commit fixes this

**Review by jeanm-m — APPROVED** [review]

**Pull request merged**